### DISCUSSION TITLE
Sites that do not support RSS

### DISCUSSION THREAD

**Discussion by exeupx** [FreshRSS/FreshRSS#5896] [Q&A] [answered]
Hello everyone
If the place for inquiry is not appropriate, I am sorry
I want to add a website, but it does not support the RSS service
Like imdb
So how do I add such sites?
Sorry, my language is not good. I use Google translator

**Comment by Alkarex** [comment, marked as answer]
For sites, which do not expose RSS / ATOM feeds:

1) You can try to write to them to motivate them to offer one.

2) If the site is using a known CMS, there might be a hidden RSS / ATOM, which you can try to guess or search. Some sites also have an official API.

3) If the site is some simple HTML (try with `curl` first) and if you code a tiny bit (XPath), you can try our built-in [HTML+XPath Web scraping functionality](https://freshrss.github.io/FreshRSS/en/users/11_website_scraping.html)

4) Otherwise, you can try systems such as the following ones, which are acting as bridges:
* [RSS-Bridge](https://github.com/RSS-Bridge/rss-bridge) (can be self-hosted)
* [RSS-Hub](https://docs.rsshub.app/) (can be self-hosted)
* [FiveFilters](https://www.fivefilters.org/full-text-rss/)
* [More alternatives...](https://alternativeto.net/software/rss-bridge--the-rss-feed-for-websites-missing-it/)

5) If you do not find the Web site you want in the above bridge systems, you can try to write a plug-in to add to their collection.

6) [Search a bit what others are doing](https://www.reddit.com/r/rss/comments/s1w9wi/is_it_possible_to_have_an_rss_for_imdb_news/) (watch out that older solutions might not be still working, as modern Web sites have a tendency to break everything every few months)

7) If none of that works, move to other services that offer better services for their users

**Comment by exeupx** [comment]
Thank you for the answer
